ports/atmel-samd/common-hal/microcontroller/Pin.c

```diff
@@ -53,6 +57,11 @@ void reset_all_pins(void) {
     pin_mask[0] &= ~(PORT_PA31);
```

Comment:
I know this code has not changed, but the `#ifdef SAMD21` looks redundant. Wasn't `PORT_PA31` already cleared?

Reply:
Ya, I removed it. I don't want to mess with the SWD lines if I can help it.